db/seeds.rb

```diff
@@ -27,6 +27,104 @@
 GradeLevel.create "title" => "College Senior", "level" => 16, "abbreviation" => "R"
 GradeLevel.create "title" => "College 5th-Year", "level" => 17
 
+# Help Texts
+HelpText.create({
+  "object_class" => "Participant", 
+  "attribute_name" => "household_size",
+  "hint" => "Include yourself in this count",
+  "audience" => "Participant"
+})
+HelpText.create({
+  "object_class" => "Mentor", 
+  "attribute_name" => "household_size",
+  "hint" => "Include the student in this count",
+  "audience" => "Mentor"
+})
+HelpText.create({
+  "object_class" => "Participant",
+  "attribute_name" => "immigrant",
+  "title" => "Do you identify yourself as an immigrant to the United States?",
+  "audience" => "Participant"
+})
+HelpText.create({
+  "object_class" => "Mentor",
+  "attribute_name" => "immigrant",
+  "title" => "Identifies as an immigrant to the United States",
+  "audience" => "Mentor"
+})
+HelpText.create({
+  "object_class" => "Participant",
+  "attribute_name" => "first_generation",
+  "title" => "Does one or more of your parents/legal guardians hold a bachelor’s/4 year degree?",
```

Comment:
Yes it has to do with `radio`, `boolean` or `checkbox` because I tried doing a `select` and it worked! So it's not an issue from our `HelpText` model.